src/vs/platform/windows/electron-main/windowsMainService.ts

```diff
@@ -884,6 +884,15 @@ export class WindowsMainService extends Disposable implements IWindowsMainServic
 
 		pathsToOpen.push(...coalesce(resolvedCliPaths));
 
+		if (cli['use-editor']) {
+			const override = cli['use-editor'];
+			for (const file of pathsToOpen) {
+				if (((file.type ?? 0) & FileType.File) === 0) { continue; }
+				// temporarily cast to any to drop readonly modifier of options
+				(file as any).options = { ...file.options, override };
```

Comment:
Do you mean the `as any` part? This is for bypassing the `readonly` modifier on `options` so that I can reassigned the field. Otherwise it has to be written in a more verbose manner as discussed in last revision https://github.com/microsoft/vscode/pull/206962#discussion_r1516504278

Reply:
Sorry, I should have followed up -- let's go either with the verbose option, or you can make it mutable by casting it via our `Mutable` utility type

Reply:
Fixed this one.